cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj

```diff
@@ -23,6 +23,7 @@
     <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <InputDir>$(SolutionDir)uwp\10.0</InputDir>
+    <InterpreterDir Condition="'$(InterpreterDir)' == ''">..\..\..\PCBuild\win32</InterpreterDir>
```

Comment:
This is the pyuwpsdk project --  the project that builds the VSIX.  Our python UWP VSIX contains 6 PythonUWP configurations ( [x86|x64|arm] x [Debug|Release]) and 1 win32/x86 python.  The win32 python which is used for intellisense and package downloading only.  The "InterpreterDir" is used for the one and only win32 python interpreter.  Hence, it doesn't need to be under "win32" condition.